### ISSUE TITLE
Status of NY Truck Routes

### ISSUE THREAD

**Issue by vdeane** [TravelMapping/HighwayData#505]
Should the NY truck routes be in the system?  NYSDOT Main Office doesn't consider them official and has no inventory of them.  They're signed by a mix of municipalities and the Regions and I'm not aware of an inventory at the regional level either.  Signage isn't consistent either; while some are the traditional bannered "Truck NY (or US) XX", others are "Truck Route (NY XX)" (as is the case for NY 14 in Elmira, NY 17C in Waverly, NY 32 in Glens Falls, and NY 340; the route number only appears on the first sign directing trucks on the route and the other turns and reassurance shields are simply "Truck Route") or simply "Truck Route".

If the routes are being kept, there are two (not including the "Truck Route" ones mentioned earlier) that I know of that aren't in the system:
-Truck NY 25 (Greenport): Moores Lane and North Road (CR 48)
-Truck NY 27 (Brooklyn): 10th Street, McDonald Avenue, Church Avenue

**Comment by jteresco** [comment]
We've discussed this once or twice before, but I don't think there was a consensus on how to proceed.  Since the routes don't seem to have any official status and signing is not always consistent, I'd be in favor of dropping them from the project.

**Comment by vdeane** [comment]
I noticed the the NY 30 one already in the system is signed with the "Truck Route" banner instead of the "Truck" banner, as is the one I mentioned on NY 32 (I'll edit the issue to reflect that).  I verified that the other ones in the system and the NY 25 and NY 27 ones I reported use "Truck".

Also: I saw on Facebook a mention of another Truck NY 25 bypassing Laurel and connecting to the Greenport one via CR 48, but I can't find signage for it on street view.

**Comment by yakra** [comment]
NY 25 Truck missing
http://tm.teresco.org/forum/index.php?topic=2125

**Comment by yakra** [comment]
From @vdeane [on the forum](http://tm.teresco.org/forum/index.php?topic=2125.msg6973#msg6973):
>There's also another Truck NY 25 in Greenport, using Moores Lane and North Road.
>
>Some others we're missing:
-Truck NY 14 (Elmira): Clemens Center Parkway, Grand Central Avenue, McCanns Boulevard
-Truck NY 17C (Waverly): Broad Street Extension, William Donnelly Indus Parkway (not sure if this one counts; Bing streetside only shows "Truck Route" signs, not "Truck NY 17C" or "Truck Route NY 17C", but I've heard Region 9 refer to it as Truck NY 17C before)
-Truck NY 27 (Brooklyn): 10th Street, McDonald Avenue, Church Avenue
-Truck NY 32 (Glens Falls): Oakland Avenue
-Truck NY 340: William Street, Main Street

**Comment by Duke87ofST** [comment]
The muck on this issue has been kicked up again by another user report of a truck route not mapped in usany: http://forum.travelmapping.net/index.php?topic=2729.0

I would favor removing any and all truck routes from usany on the grounds that they are not officially recognized by NYSDOT as part of the state touring route system.

**Comment by jteresco** [comment]
I remain in favor of dumping routes like these in usany and anywhere else that has a similar situation.

**Comment by vdeane** [comment]
Seems like they're currently popping up everywhere in usapa.

**Comment by rickmastfan67** [comment]
As far as I understand, all the PA ones are being posted by PennDOT.

**Comment by Markkos1992** [comment]
Yes the PA ones are being posted by PennDOT.  I was putting them in via a "in as I go" approach. 

They were already (to a minor extent) being put in before I took over PA so I believed that I was basically put on the side of putting these in anyhow.

Having said that, there are weight-restricted bridges that do not have them.  I found one on PA 191 just past the end of its PA 447 duplex that did not have one.

All weight restricted bridges can be found here:  https://pennshare.maps.arcgis.com/apps/webappviewer/index.html?id=5fcd857b5874471f8cda56337566861f
